### PR TITLE
Fixed the POM version from '1.0.0-b6-SNAPSHOT' to '1.0.0-SNAPSHOT'

### PR DESCRIPTION
**Pull request by dearrudam**
Ref: #308

### CODE REVIEW

**Comment by genie-jnosql** [comment]
Can one of the admins verify this patch?

**Comment by dearrudam** [comment]
@otaviojava could you review this PR, please?

**Comment by otaviojava** [comment]
Hey @dearrudam, this PR will merge later this week: https://github.com/eclipse/jnosql/pull/309

Do you mind rebasing it?

**Comment by dearrudam** [comment]
@otaviojava no problem! I'll be waiting for #309 merging then! Thanks for letting me know! 🙂👍

**Comment by dearrudam** [comment]
@otaviojava no problem! I'll be waiting for #309 merging then! Thanks for letting me know! 🙂👍

**Comment by otaviojava** [comment]
@dearrudam please go ahead

File: jnosql-mapping/jnosql-mapping-graph/pom.xml [hunk 1]
```diff
@@ -31,15 +31,18 @@
...
         <dependency>
             <groupId>org.eclipse.jnosql.communication</groupId>
             <artifactId>jnosql-communication-query</artifactId>
-            <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>jnosql-mapping-core</artifactId>
-            <version>${project.version}</version>
+            <groupId>jakarta.nosql.tck.mapping</groupId>
```
Comment (otaviojava) [inline jnosql-mapping/jnosql-mapping-graph/pom.xml:43]: @dearrudam, the TCK does not exist anymore, at least yet.

File: jnosql-mapping/jnosql-jupiter/pom.xml [hunk 2]
```diff
@@ -35,7 +35,6 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
-            <version>${junit.version}</version>
```
Comment (otaviojava) [inline jnosql-mapping/jnosql-jupiter/pom.xml:38]: @dearrudam if you remove this version line ti won't work.

File: jnosql-mapping/jnosql-mapping-graph/pom.xml [hunk 3]
```diff
@@ -31,15 +31,18 @@
         <neo4j.connector.version>0.9-3.4.0</neo4j.connector.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.eclipse.jnosql.mapping</groupId>
```
Comment (otaviojava) [inline jnosql-mapping/jnosql-mapping-graph/pom.xml:35]: @dearrudam, I would remove it.
I would change the necessary for this PR.

File: jnosql-mapping/jnosql-mapping-graph/pom.xml (hunk 1 above)
Comment (dearrudam) [reply jnosql-mapping/jnosql-mapping-graph/pom.xml:43]: @otaviojava could I remove the TCK dependencies from the `dependencyManagement` tag in the `jnosql-parent`'s POM?

File: jnosql-mapping/jnosql-jupiter/pom.xml (hunk 2 above)
Comment (dearrudam) [reply jnosql-mapping/jnosql-jupiter/pom.xml:38]: @otaviojava this dependency is managed by the `jnosql-parent`'s POM... the `mvn -B package` command works as expected, we can see it by taking a look at the check results [here](https://github.com/eclipse/jnosql/pull/310/checks)

File: jnosql-mapping/jnosql-mapping-graph/pom.xml (hunk 1 above)
Comment (otaviojava) [reply jnosql-mapping/jnosql-mapping-graph/pom.xml:43]: Yes, it does not exist anymore.

Comment (dearrudam) [reply jnosql-mapping/jnosql-mapping-graph/pom.xml:43]: @otaviojava okay! thanks! I'm going to remove it!

File: pom.xml [hunk 4]
```diff
@@ -61,117 +56,490 @@
...
+                <groupId>org.eclipse.jnosql.communication</groupId>
+                <artifactId>jnosql-communication-key-value</artifactId>
+                <version>${jakarta.nosql.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.nosql.mapping</groupId>
+                <artifactId>mapping-key-value</artifactId>
+                <version>${jakarta.nosql.version}</version>
+            </dependency>
+            <!-- TCKs -->
```
[the remark this answers is not in the document]
Comment (dearrudam) [reply pom.xml:255]: @otaviojava okay! I'm doing it right away!

File: pom.xml [hunk 5]
```diff
@@ -61,117 +56,490 @@
...
+                            <goal>prepare-agent</goal>
+                        </goals>
+                        <configuration>
+                            <destFile>${sonar.jacoco.reportPath}</destFile>
+                            <append>true</append>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
```
Comment (otaviojava) [inline pom.xml:422]: @dearrudam please, don't forget the maven-gpg-plugin here as well.

https://github.com/eclipse-ee4j/nosql/blob/main/pom.xml#L327L346

File: jnosql-mapping/jnosql-jupiter/pom.xml (hunk 2 above)
Comment (otaviojava) [reply jnosql-mapping/jnosql-jupiter/pom.xml:38]: ahhh ok, thanks :)

File: pom.xml (hunk 5 above)
Comment (dearrudam) [reply pom.xml:422]: @otaviojava it is on the `jnosql-parent`'s POM under the `release` profile already. Do I need to remove it?!

File: jnosql-mapping/jnosql-mapping-graph/pom.xml (hunk 3 above)
Comment (dearrudam) [reply jnosql-mapping/jnosql-mapping-graph/pom.xml:35]: @otaviojava I figure out that `org.eclipse.jnosql.mapping.test.jupiter` contains classes pretty much similar to provided ones by the `jnosql-junit` module ... could I refactor it to use the `jnosql-junit` module as test dependency?

**Comment by dearrudam** [comment]
@otaviojava please, go ahead!

File: jnosql-mapping/jnosql-mapping-graph/pom.xml (hunk 3 above)
Comment (dearrudam) [reply jnosql-mapping/jnosql-mapping-graph/pom.xml:35]: @otaviojava I did it! :smile:

Comment (otaviojava) [reply jnosql-mapping/jnosql-mapping-graph/pom.xml:35]: Thanks :)

**Comment by otaviojava** [comment]
Great job @dearrudam merged

**Comment by otaviojava** [comment]
@dearrudam do you want to do the same here?
https://github.com/eclipse-ee4j/nosql/blob/main/pom.xml

**Pull request merged**